### PR TITLE
Create example.jsonld

### PR DESCRIPTION
**Pull request by bs524**

### CODE REVIEW

File: WaterConsumption/examples/example.jsonld [hunk 1]
```diff
@@ -0,0 +1,138 @@
+{
```
Comment (albertoabellagarcia) [inline WaterConsumption/examples/example.jsonld]: this should be name example-normalized.jsonld.
example.jsonld is for key-values format

File: WaterConsumption/examples/example.jsonld [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    "id": "urn:ngsi-ld:Consumer:Consumer139E643",
+    "type": "Consumer",
+    "https://ontology.eglobalmark.com/WaterSmartMeter#acquisitionStageFailure": {
```
Comment (albertoabellagarcia) [inline WaterConsumption/examples/example.jsonld]: The names of the keys should not be long iri (they depend on the context but the short ones). In this case, the context is a specific one, not a general one.

File: WaterConsumption/examples/example.jsonld [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        "type": "Property",
+        "observedBy": {
```
Comment (albertoabellagarcia) [inline WaterConsumption/examples/example.jsonld]: Although accepted the observedAt and observedBy does not provide much value to be repeated in all properties.

File: WaterConsumption/examples/example.jsonld [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:139E643"
+        },
+        "value": 191051,
+        "observedAt": "2021-05-23T23:14:16.000Z",
+        "unitCode": "LTR"
+    },
+    "@context": [
```
Comment (albertoabellagarcia) [inline WaterConsumption/examples/example.jsonld]: It will be changed this context to a more general one

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
Please attend comments

File: WaterConsumption/examples/example.jsonld (hunk 1 above)
Comment (bs524) [reply WaterConsumption/examples/example.jsonld]: Changed

File: WaterConsumption/examples/example.jsonld (hunk 3 above)
Comment (bs524) [reply WaterConsumption/examples/example.jsonld]: From what I was told, we (Exeter) did not create the waterconsumption object and therefore is hesistant to change its set-up.

**Review by albertoabellagarcia — APPROVED** [review]

File: WaterConsumption/examples/example-normalized.jsonld [hunk 5]
```diff
@@ -0,0 +1,138 @@
+{
```
Comment (albertoabellagarcia) [inline WaterConsumption/examples/example-normalized.jsonld:1]: could you remove the long url from the property names?

**Review by albertoabellagarcia — APPROVED** [review]

**Pull request merged**